src/Features/CSharp/Portable/Wrapping/SeparatedSyntaxList/CSharpParameterWrapper.cs

```diff
@@ -52,7 +50,13 @@ protected override SeparatedSyntaxList<ParameterSyntax> GetListItems(BaseParamet
             var lastToken = listSyntax.GetLastToken();
 
             var headerSpan = TextSpan.FromBounds(firstToken.SpanStart, lastToken.Span.End);
-            return headerSpan.IntersectsWith(position);
+            if (!headerSpan.IntersectsWith(position))
+                return false;
+
+            if (containsSyntaxError && ContainsOverlappingSyntaxErrror(declaration, headerSpan))
```

Comment:
so it looks like here this is checking if the syntax error overlaps with the method header and bailing out if it does.  But based on the test and issue comments it seems like the desired behavior is if it overlaps with the header or anything prior to the header.  Is that check being done somewhere else?

And would it be possible to have a test where there is an error prior to the method to confirm it bails out (unless it already exists)

Reply:
discussed offline - I misunderstood the intention :)